source/common/config/utility.h

```diff
@@ -41,6 +41,9 @@ namespace Config {
 
 constexpr absl::string_view Wildcard = "*";
 
+static constexpr uint64_t RetryBaseIntervalMs = 500;
```

Comment:
Are these global constants GRPC retry mechanism specific?  Need some comments here. Also is here the best place to define them?

Reply:
These are not intended to be specific to GRPC retry timers but rather the the default time for @prepareBackoffStrategy method in this file.